corral/bundle/bundle.pony

```diff
@@ -7,25 +7,24 @@ primitive BundleFile
   """
   Loader and creator of Bundle files.
   """
-  fun find_bundle_dir(env: Env, log: Log): (FilePath | None) =>
-    let cwd = Path.cwd()
-    var dir = cwd
-    while dir.size() > 0 do
-      log.fine("Looking for " + Files.bundle_filename() + " in: '" + dir + "'")
+  fun find_bundle_dir(env: Env, path: String, log: Log): (FilePath | None) =>
+    var path' = path
+    while path'.size() > 0 do
+      log.info("Looking for " + Files.bundle_filename() + " in: '" + path' + "'")
       try
-        let dir_path = FilePath(env.root as AmbientAuth, dir)?
-        let bundle_file = dir_path.join(Files.bundle_filename())?
+        let dir = FilePath(env.root as AmbientAuth, path')?
```

Comment:
I had named this `dir_path` to make it clear that it was a FilePath object and not just a string.

Reply:
Makes sense, just reverted the change.